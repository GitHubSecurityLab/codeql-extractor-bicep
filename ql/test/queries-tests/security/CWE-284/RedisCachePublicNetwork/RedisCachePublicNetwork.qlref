security/CWE-284/RedisCachePublicNetwork.ql
