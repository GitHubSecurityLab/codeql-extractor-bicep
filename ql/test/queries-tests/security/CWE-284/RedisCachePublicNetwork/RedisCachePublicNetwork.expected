| app.bicep:13:1:19:1 | RedisCacheResource | Redis Cache 'public-redis' has public network access enabled, exposing it to the internet. |
